cleaners/internet_explorer.xml

```diff
@@ -41,19 +53,29 @@
     <label>History</label>
     <description>List of visited web pages</description>
     <action command="delete" search="walk.files" path="%USERPROFILE%\AppData\Local\Microsoft\Windows\History\"/>
-    <action command="delete" search="walk.files" path="%USERPROFILE%\Local Settings\History\"/>
     <action command="delete" search="walk.files" path="%LOCALAPPDATA%\Microsoft\Internet Explorer\Recovery\Active\"/>
     <action command="delete" search="walk.files" path="%LOCALAPPDATA%\Microsoft\Internet Explorer\Recovery\Immersive\Active\"/>
     <action command="delete" search="walk.files" path="%LOCALAPPDATA%\Microsoft\Internet Explorer\Recovery\Last Active\"/>
     <!-- "How to clear the History entries in Internet Explorer", http://support.microsoft.com/kb/157729 -->
     <action command="winreg" path="HKCU\Software\Microsoft\Internet Explorer\TypedURLs"/>
+    <!-- Used e.g. in Windows XP (and maybe before on NT) - English-only-Support: -->
+    <action command="delete" search="walk.files" path="%USERPROFILE%\Local Settings\History\"/>
+    <!-- Used e.g. in Windows XP (and maybe before on NT) - some none-English-Support: -->
+    <!-- Polish Windows XP Support: -->
+    <action command="delete" search="walk.files" path="%USERPROFILE%\Ustawienia lokalne\History\"/>
   </option>
   <option id="temporary_files">
     <label>Temporary files</label>
     <description>Delete the temporary files</description>
     <action command="delete" search="walk.files" path="%APPDATA%\Microsoft\Windows\IETldCache\"/>
     <action command="delete" search="walk.files" path="%USERPROFILE%\AppData\Local\Microsoft\Windows\Temporary Internet Files\"/>
-    <action command="delete" search="walk.files" path="%USERPROFILE%\Local Settings\Temporary Internet Files\"/>
     <action command="delete" search="walk.files" path="%LOCALAPPDATA%\Microsoft\Feeds Cache\"/>
+    <!-- Used e.g. in Windows XP (and maybe before on NT) - English-only-Support: -->
+    <action command="delete" search="walk.all" path="%USERPROFILE%\Local Settings\Temporary Internet Files\"/>
+    <action command="delete" search="glob" path="%USERPROFILE%\Local Settings\Temporary Internet Files\*"/>
```

Comment:
The glob search on line 75 will match all files directly in this folder, while the action above (line 74) already deleted all contents recursively, so line 75 will do nothing

Reply:
I hope I get it now finally when and how to use `glob`! :-)